xcell/mappers/mapper_BOSSCMASS.py

```diff
@@ -0,0 +1,37 @@
+from .mapper_base import MapperSDSS
+import healpy as hp
+
+
+class MapperBOSSCMASS(MapperSDSS):
+    def __init__(self, config):
+        """
+        config - dict
+          {'data_catalogs':['eBOSS_QSO_clustering_data-NGC-vDR16.fits'],
+           'random_catalogs':['eBOSS_QSO_clustering_random-NGC-vDR16.fits'],
+           'z_edges':[0, 1.5],
+           'nside':nside,
+           'nside_mask': nside_mask,
+           'mask_name': 'mask_QSO_NGC_1'}
+        """
+        self._get_defaults(config)
```

Comment:
A lot of these lines are actually common to all the different subclasses. How about just put the common ones in `MapperSDSS` and then call 
```
super(MapperBOSSCMASS, self).__init__(config)
```
and add specific lines afterwards

Reply:
done through get_SDSS_defaults method just as with get_defaults in mapper_base